app/models/hbx_enrollment.rb

```diff
@@ -1523,9 +1523,16 @@ def decorated_elected_plans(coverage_kind, market = nil)
     filtered_elected_plans(elected_plans).collect {|plan| UnassistedPlanCostDecorator.new(plan, self)}
   end
 
-  def filtered_elected_plans(elected_plans)
-    elected_plans.reject!(&:allows_child_only_offering?) if ::EnrollRegistry.feature_enabled?(:exclude_child_only_offering) && any_member_greater_than_18?
-    elected_plans.reject!(&:allows_adult_and_child_only_offering?) if ::EnrollRegistry.feature_enabled?(:exclude_adult_and_child_only_offering) && !any_member_greater_than_18?
+  def filtered_elected_plans(elected_plans, coverage_kind)
+    # TODO: Address this more structurally later.  The logic for pediatric
+    #       dental plan eligibilty is not encoded in a way that interacts
+    #       intuitively with the two available values for whether a plan is
+    #       'child only' or 'adult and child only' on the child_only_offering
+    #       property of BenefitMarkets::Products::Product.
+    if coverage_kind == 'dental'
+      elected_plans.reject(&:allows_child_only_offering?) if ::EnrollRegistry.feature_enabled?(:exclude_child_only_offering) && any_member_greater_than_18?
```

Comment:
this should to be `reject!` instead of `reject`

Reply:
Yup